frontend/src/views/SqlEditor/AsidePanel/DatabaseTree.vue

```diff
@@ -67,6 +90,45 @@ const { setConnectionContext } = useNamespacedActions<SqlEditorActions>(
   ["setConnectionContext"]
 );
 
+const searchPattern = ref();
+const showDropdown = ref(false);
+const x = ref(0);
+const y = ref(0);
+const selectedKeys = ref<string[] | number[]>([]);
+const sheetContext = ref<DropdownOption>();
+const dropdownOptions = computed(() => {
+  if (!sheetContext.value) {
+    return [];
+  }
+  if (sheetContext.value.type === "table") {
+    return [
+      {
+        label: "Alter table",
+        key: "alter-table",
+        item: sheetContext.value,
+      },
+      // TODO Just a thought
+      // {
+      //   label: "Copy name",
+      //   key: "copy-name",
+      //   item: sheetContext.value,
+      // },
+    ];
+  }
+  return [
+    {
+      label: "Open in new tab",
+      key: "open-in-new-tab",
```

Comment:
for key like this, use reverse domain name notation is more extensible https://en.wikipedia.org/wiki/Reverse_domain_name_notation

editor.sheet.new

This allows to have a more structured action set (we may have editor.sheet.select later)

Reply:
TIL